lib/app/pages/welcome_page/welcome_page.dart

```diff
@@ -87,8 +87,9 @@ class _WelcomePageState extends State<WelcomePage>
               if (state is CreatedActivityState) {
                 Navigator.pushReplacementNamed(context, HOME_PAGE,
                     arguments: state.activity);
-              // } else if (state is HasLastActivityState) {
-              //   _activityBloc.add(GetActivityEvent());
+                _activityBloc.add(GetActivityEvent());
+                // } else if (state is HasLastActivityState) {
```

Comment:
Show unblock and retry because this use to go the last activity If the users created their activity and have not deleted that.